pkg/virt-controller/watch/migration.go

```diff
@@ -1397,6 +1390,33 @@ func (c *MigrationController) getNodeForVMI(vmi *virtv1.VirtualMachineInstance)
 	return node, nil
 }
 
+func (c *MigrationController) alertIfHostModelIsUnschedulable(vmi *virtv1.VirtualMachineInstance, targetPod *k8sv1.Pod) {
+	const warningMessage = "Migration cannot proceed since no node is suitable to run the required CPU model / required features"
+	fittingNodeFound := false
+
+	if cpu := vmi.Spec.Domain.CPU; cpu != nil && cpu.Model == virtv1.CPUModeHostModel && isPodPending(targetPod) {
+		nodes := c.nodeInformer.GetStore().List()
+		for _, nodeInterface := range nodes {
+			node := nodeInterface.(*k8sv1.Node)
+
+			if node.Name == vmi.Status.NodeName {
+				continue // avoid checking the VMI's source node
+			}
+
+			if isNodeSuitableForHostModelMigration(node, targetPod) {
+				log.Log.Object(vmi).Infof("Node %s is suitable to run vmi %s host model cpu mode (more nodes may fit as well)", node.Name, vmi.Name)
```

Comment:
By "translation" you mean the host CPU model which we're trying to match to?
I thought of that, the thing is that it can be very verbose, as CPU required features are also crucial here and there could be many of them. On the other hand this surely can be helpful for debugging.

No problem, I'll add that